ros2_batch_job/osx_batch/__init__.py

```diff
@@ -38,6 +38,8 @@ def pre(self):
             os.environ['LANG'] = 'en_US.UTF-8'
         if 'ROS_DOMAIN_ID' not in os.environ:
             os.environ['ROS_DOMAIN_ID'] = '111'
+        if 'OPENSSL_ROOT_DIR' not in os.environ:
+            os.environ['OPENSSL_ROOT_DIR'] = '/usr/local/Cellar/openssl/1.0.2k'
```

Comment:
as mentioned in https://github.com/ros2/ci/pull/76#discussion_r121731426, I'd expect that to be done directly in the "bashrc" of the build machines rather than in this script. Didn't have time to do / test it though.